warehouse/query-core/src/main/java/datawave/query/jexl/visitors/IteratorBuildingVisitor.java

```diff
@@ -1032,20 +1032,6 @@ public Object visit(ASTNumberLiteral node, Object o) {
         return null;
     }
     
-    private boolean isUsable(Path path) throws IOException {
-        try {
-            if (!hdfsFileSystem.getFileSystem(path.toUri()).mkdirs(path)) {
-                throw new IOException("Unable to mkdirs: fs.mkdirs(" + path + ")->false");
-            }
-        } catch (MalformedURLException e) {
-            throw new IOException("Unable to load hadoop configuration", e);
-        } catch (Exception e) {
-            log.warn("Unable to access " + path, e);
-            return false;
-        }
-        return true;
-    }
-    
     /**
      * Create a cache directory path for a specified regex node. If alternatives have been specified, then random alternatives will be attempted until one is
```

Comment:
I think this comment needs to be updated. This method is no longer creating new directories if they do not exist, it is now building up a list of potential hdfs directories based on the config.

Reply:
Comment has been updated.